velox/docs/functions/spark/array.rst

```diff
@@ -125,13 +125,34 @@ Array Functions
 .. spark:function:: array_sort(array(E)) -> array(E)
 
     Returns an array which has the sorted order of the input array(E). The elements of array(E) must
-    be orderable. Null elements will be placed at the end of the returned array. ::
+    be orderable. Null/NaN elements will be placed at the end of the returned array. ::
```

Comment:
> Null/NaN elements will be placed at the end

For floating point data, which will be put at the end if null and NaN both exist?

Reply:
NaN is before NULL, there's a test already covers this, `floatingPointInput` in `ArraySortTestData`